cg/constants/nf_analysis.py

```diff
@@ -21,6 +21,7 @@ class NfTowerStatus(StrEnum):
     "MEDIAN_TARGET_COVERAGE": {"norm": "gt", "threshold": 26},
     "PCT_TARGET_BASES_10X": {"norm": "gt", "threshold": 0.95},
     "PCT_EXC_ADAPTER": {"norm": "lt", "threshold": 0.0005},
+    "predicted_sex_sex_check": {"norm": "eq", "threshold": None},
```

Comment:
```suggestion
    "sex_check_predicted_sex": {"norm": "eq", "threshold": None},
```

Reply:
I agree with you, but this is what the variable is called in the multiqc json output file, so I will have to keep it as is.

Reply:
I was afraid of that, 👍 .